dali/base/dautils.cpp

```diff
@@ -515,7 +523,17 @@ void CDfsLogicalFileName::set(const char *name, bool removeForeign)
         lfn.set(full);
         return;
     }
-    normalizeName(name, lfn, false);
+    if (isExternalFile(name))
+    {
+        lfn.set(name);
+        lfn.toLowerCase();
```

Comment:
It's okay to lower case if it's an external (consistent with previous behaviour), but not if it's a query ('>').
On detection of query ('>') old code just appends original text.

Also by doing a lfn.set(name) here, it's losing some of the formatting it had before, i.e. it used to strip blank spaces and also use getUrlStr to format the endpoint.
It may not cause any problems in practice, if there aren't any irregular filenames out there in use, but it would be safer to keep it as it was.